dts/bindings/wifi/nordic,nrf7002-qspi.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: This is a representation of the nRF7002 WiFi chip.
+
+compatible: "nordic,nrf7002-qspi"
+
+include: "nordic,nrf7002.yaml"
+
+on-bus: qspi
+
+properties:
+  sck-frequency:
+    type: int
+    required: true
+    description: |
+      Maximum clock speed supported by the device, in Hz.
+
+  reg:
+    required: true
+
+  label:
+    required: false
```

Comment:
Remove this; it is unnecessary if you include base.yaml appropriately as @gmarull has recommended you should do.

Reply:
Sure, `base.yaml` is already included in the `nordic,nrf7002.yaml` which is included in this file.